lib/QoreClass.cpp

```diff
@@ -2889,6 +2889,11 @@ void BCSMList::resolveCopy() {
    }
 }
 
+QoreClass::QoreClass(const char* nme, int64 dom) : priv(new qore_class_private(this, nme, dom)) {
+   priv->orNothingTypeInfo = new OrNothingTypeInfo(*(priv->typeInfo), nme);
+   priv->owns_ornothingtypeinfo = true;
+}
+
 QoreClass::QoreClass(const char* nme, int dom) : priv(new qore_class_private(this, nme, dom)) {
```

Comment:
no, this is part of the ABI, so it must stay in the library.  It's been deprecated and marked as `explicit` in the header so that the `int64` version will be used instead